salt/cleanup/remove-pre-reboot.sls

```diff
@@ -2,24 +2,10 @@
 # try to remove some dirs that could contain sensitive
 # information, even when they were not directly managed by us
 
-wipe-etc-kubernetes:
-  cmd.run:
-    - name: rm -rf /etc/kubernetes/*
```

Comment:
I would keep it, in case if CaaSP will be deployed with CPI it will contains Public/OpenStack Cloud credentials